examples/jetstream/limits-stream/node/main.js

```diff
@@ -0,0 +1,97 @@
+import {connect, StringCodec} from "nats";
+
+// Get the passed NATS_URL or fallback to the default. This can be
+// a comma-separated string.
+const servers = process.env.NATS_URL || "nats://localhost:4222";
+
+// Create a client connection to an available NATS server.
+const nc = await connect({
+  servers: servers.split(","),
+});
+
+// NATS message payloads are byte arrays, so we need to have a codec
+// to serialize and deserialize payloads in order to work with them.
+// Another built-in codec is JSONCodec or you can implement your own.
+const sc = StringCodec();
+
+// Access the JetStream manager which provides the methods for managing
+// streams and consumers.
+const jsm = await nc.jetstreamManager();
+
+// Declare the initial stream config. A stream can bind one or more
+// subjects that are not overlapping with other streams. By default,
+// a stream will have one replica and use file storage.
+const cfg = {
+  name: "EVENTS",
+  subjects: ["events.>"],
+};
+
+// Add/create the stream.
+await jsm.streams.add(cfg)
+console.log("created the stream")
+
+// Access the JetStream client for publishing and subscribing to streams.
+const js = nc.jetstream();
+
+// Publish a series of messages and wait for each one to be completed.
+await js.publish("events.page_loaded");
+await js.publish("events.mouse_clicked");
+await js.publish("events.mouse_clicked");
+await js.publish("events.page_loaded");
+await js.publish("events.mouse_clicked");
+await js.publish("events.input_focused");
+console.log("published 6 messages");
+
+const events = [
+  "events.input_changed",
+  "events.input_blurred",
+  "events.key_pressed",
+  "events.input_focused",
+  "events.input_changed",
+  "events.input_blurred",
+];
+
+// Publish a series of events and wait for _all_ of the publishes
+// have resolved.
+const batch = [];
```

Comment:
just a nitpick - you can leave as you have it, but if you have a list:

const proms = events.map((e) => { return js.publish(e); });
await Promise.all(proms);